frontend/src/components/Cases/AccordionItem.js

```diff
@@ -0,0 +1,199 @@
+import AccordionSvg from "./AccordionSvg";
+import { schemaDisplay, copyToClipboard } from "../../utilities/utilFunctions";
+import Svg from "../../assets/svg/Svg";
+
+const AccordionItem = ({ eachCase, implementations, caseImplementation }) => {
+  const seq = eachCase.seq;
+  const description = eachCase.case.description;
+  const schema = eachCase.case.schema;
+  const tests = eachCase.case.tests;
+
+  function result(index, implementation) {
+    var testResult;
```

Comment:
Any reason why you use `var` in some places? Checked few of them and looks like they can be replaced by `let` or `const`. I think that as a general rule of thumb the latter two should be used - they are much more intuitive in terms of scope and visibility

Reply:
No. I'll change then to `let`/`const`

Reply:
No. I'll change them to `let`/`const`